src/main/scala/io/opentargets/etl/backend/drug/Molecule.scala

```diff
@@ -64,7 +64,9 @@ object Molecule extends LazyLogging {
         col("molecule_chembl_id").as("id"),
         col("molecule_structures.canonical_smiles").as("canonicalSmiles"),
         col("molecule_structures.standard_inchi_key").as("inchiKey"),
-        col("molecule_type").as("drugType"),
+        when(col("molecule_type").isNotNull, col("molecule_type"))
```

Comment:
@JarrodBaker when the when section is about null check using coalesce makes things clearer. I suggest coalescing with lit

Reply:
Updated use `coalesce`